js/src/dapps/registry/Events/events.js

```diff
@@ -162,33 +241,55 @@ export default class Events extends Component {
   }
 
   onReservedToggle = (e, isToggled) => {
-    const { pending, subscriptions, actions } = this.props;
+    const { pending, subscriptions, subscribe, unsubscribe } = this.props;
```

Comment:
It would make sense following the variable definitions with a blank line before the code.